### DISCUSSION TITLE
Benchmark time per epoch for the example dataset?

### DISCUSSION THREAD

**Discussion by Schobs** [MIC-DKFZ/nnUNet#746] [Q&A] [answered]
Hi,

I'm trying nn-UNet on the hippocampus example dataset you suggest, using only the command you suggested. Each epoch is taking around ~38 seconds. What is a good benchmark for this? What kind of times are other people achieving with their set up? For reference, I have an RTX 3080 mobile.

**Comment by FabianIsensee** [comment, marked as answer]
Hi,
no idea tbh. Your number is slower than a 2080ti desktop, that would worry me a bit. I really don't know what the mobile 3080 is supposed to deliver though
Best,
Fabian
